packages/backend/src/analytics/LightdashAnalytics.ts

```diff
@@ -452,6 +455,22 @@ type AnalyticsDashboardView = BaseTrack & {
     };
 };
 
+type PinnedListUpdated = BaseTrack & {
+    event: 'pinned_list.updated';
+    userId: string;
+    properties: {
+        projectId: string;
+        organizationId: string;
+        source: 'homepage' | Space['name'];
+        pinnedListUuid: string;
+        pinnedItems: {
+            charts: CreateChartPinnedItem[];
+            dashboards: CreateDashboardPinnedItem[];
+        };
```

Comment:
We only need to track the ids here.
```suggestion
        pinnedItems: {
            chartIds: string[];
            dashboardIds: string[];
        };
```